osc/osc.go

```diff
@@ -474,21 +468,9 @@ func (b *Bundle) MarshalBinary() ([]byte, error) {
 // specifies the IP address and `port` defines the target port where the
 // messages and bundles will be send to.
 func NewClient(ip string, port int) *Client {
-	return &Client{ip: ip, port: port, laddr: nil}
+	return &Client{IP: ip, Port: port, laddr: nil}
 }
 
-// IP returns the IP address.
-func (c *Client) IP() string { return c.ip }
-
-// SetIP sets a new IP address.
-func (c *Client) SetIP(ip string) { c.ip = ip }
-
-// Port returns the port.
-func (c *Client) Port() int { return c.port }
-
-// SetPort sets a new port.
-func (c *Client) SetPort(port int) { c.port = port }
-
```

Comment:
I could put these back in, but I don't really see a reason for them existing in the first place.